### PR TITLE
doc: translation for effective-hpa-with-prometheus-adapter.md

### LINKED ISSUE #403: Document Translation for docs/tutorials/effective-hpa-with-prometheus-adapter.zh.md
## Describe the feature
Translate document `docs/tutorials/effective-hpa-with-prometheus-adapter.zh.md` to english.
Please make the new content into `docs/tutorials/effective-hpa-with-prometheus-adapter.md`.
Ensure the render succeeds.

### LINKED ISSUE #421: Document translate for docs/tutorials/timeseriees-forecasting-by-dsp.md
Document translate, Please translate docs/tutorials/timeseriees-forecasting-by-dsp.md to english.

### PR DESCRIPTION
**Pull request by jxs1211**
Signed-off-by: shenxianjie <327411586@qq.com>

#### What type of PR is this?
translation for effective-hpa-with-prometheus-adapter.md

#### What this PR does / why we need it:
fix the issue #403 

#### Which issue(s) this PR fixes:

Fixes #403

### CODE REVIEW

**Comment by jxs1211** [comment]
@qmhu I notice that the math syntax is still not working on the web page, I try to fix it and check it in my local preview is ok and the syntax has followed the instruction below:
```
Assume that the data sampled between the  $m$ th and $n$ th sample points are missing ($m+1 < n$), and the sample values at the $m$ and $n$ points are $v_m$ and $v_n$ respectively, so that
$$
\Delta = {v_n-v_m \over n-m}
$$
, then the filled data between $m$ and $n$ are $ v_m+\Delta , v_m+2\Delta , ... $
```

File: docs/tutorials/effective-hpa-with-prometheus-adapter.md [hunk 1]
```diff
@@ -39,39 +39,39 @@ v1beta1.flowcontrol.apiserver.k8s.io   Local                             True
...
 
-在安装 PrometheusAdapter 时没有将 ApiService 指向 PrometheusAdapter，因此为了让 PrometheusAdapter 也可以提供自定义 Metric，通过 Crane Metric Adapter 的 `RemoteAdapter` 功能将请求转发给 PrometheusAdapter。
+The installation of PrometheusAdapter did not point the ApiService to PrometheusAdapter, so in order to allow PrometheusAdapter to provide custom Metric as well, the ``RemoteAdapter`` function of Crane Metric Adapter is used to forward requests to PrometheusAdapter.
 
-修改 Metric-Adapter 的配置，将 PrometheusAdapter 的 Service 配置成 Crane Metric Adapter 的 RemoteAdapter
+Modify the Metric-Adapter configuration to configure PrometheusAdapter's Service as Crane Metric Adapter's RemoteAdapter
 
 ```bash
-# 查看当前集群 ApiService
+# View the current cluster ApiService
```
Comment (qmhu) [inline docs/tutorials/effective-hpa-with-prometheus-adapter.md:70]: remove `cluster`

File: docs/tutorials/effective-hpa-with-prometheus-adapter.md [hunk 2]
```diff
@@ -219,15 +219,15 @@ kubectl get --raw /apis/custom.metrics.k8s.io/v1beta1 | jq .
 }
 ```
 
-这表明现在可以通过 Pod Metric 配置 HPA。
+This indicates that the HPA can now be configured via Pod Metric.
 
-### 配置弹性
+### Configuring resiliency
```
Comment (qmhu) [inline docs/tutorials/effective-hpa-with-prometheus-adapter.md:224]: resiliency -> autoscaling

**Review by qmhu — COMMENTED** [review]
Please use autoscaling to replace resiliency

**Comment by jxs1211** [comment]
@qmhu all the comments have been fixed.

File: docs/tutorials/effective-hpa-with-prometheus-adapter.md [hunk 3]
```diff
@@ -397,6 +397,6 @@ spec:
     name: sample-app
 ```
 
-## 总结
+## Summary
 
-由于生产环境的复杂性，基于多指标的弹性（CPU/Memory/自定义指标）往往是生产应用的常见选择，因此 Effective HPA 通过预测算法覆盖了多指标的弹性，达到了帮助更多业务在生产环境落地水平弹性的成效。
+Due to the complexity of production environments, multi-metric-based resiliency (CPU/Memory/custom metrics) is often a common choice for production applications, so Effective HPA achieves the effectiveness of helping more businesses land horizontal resiliency in production environments by covering multi-metric resiliency with predictive algorithms.
```
Comment (qmhu) [inline docs/tutorials/effective-hpa-with-prometheus-adapter.md:402]: Please also replace here

Comment (jxs1211) [reply docs/tutorials/effective-hpa-with-prometheus-adapter.md]: @qmhu fixed

**Review by qmhu — APPROVED** [review]

**Pull request merged**